pkg/framework/plugins/defaultevictor/defaultevictor_test.go

```diff
@@ -627,7 +627,9 @@ func TestDefaultEvictorFilter(t *testing.T) {
 			}
 
 			result := evictorPlugin.(framework.EvictorPlugin).Filter(test.pods[0])
-			if result != test.result {
+			// Not handling errors here since we expect them when, for example, all nodes are not available for nodefit
+			resultPreEvictionFilter, _ := evictorPlugin.(framework.EvictorPlugin).PreEvictionFilter(test.pods[0])
```

Comment:
Sooner or later it will be practical to create a separate unit test for `PreEvictionFilter`. Easier to do it now :)

Reply:
On it :smile: